bundles/framework/publisher2/view/PanelGeneralInfo.js

```diff
@@ -125,6 +144,16 @@ Oskari.clazz.define('Oskari.mapframework.bundle.publisher2.view.PanelGeneralInfo
 
             me._languageChanged(selectedLang);
         },
+        /*
+         * Function implemented by modifying function introduced in https://miguelmota.com/bytes/validate-domain-regex/
+         */
+        _isValidDomain (v) {
```

Comment:
This should be moved to Oskari.util.

Reply:
Function moved to Oskari.util and some unit tests is added.